server_api.py

```diff
@@ -1254,9 +1254,16 @@ def exchange_code_and_store_credentials(code):
 
 def get_upcoming_events(credentials):
     target_calendars = ['nctupac@gmail.com']
-    events = []
+    events = {}
     http = credentials.authorize(httplib2.Http())
     service = discovery.build('calendar', 'v3', http=http)
+    #calendars' ID and name dict
+    calendars_dict = {}
+    calendars_dict["92v3jovo2papnqrs02a4lrjs84@group.calendar.google.com"] = "Speechs"
+    calendars_dict["8jpj4urenbr3tsh0vg353fu2fo@group.calendar.google.com"] = "Activities"
```

Comment:
The above two calendar id is existed in your test google account or our sharing account??

Reply:
In the account on the trello.